backend/source/wai-nepal/forms/557950127.json

```diff
@@ -204,13 +204,13 @@
             {
               "id": 552050126,
               "options": [
-                "Piped supply inside the building /Piped water into dwelling (into the building)",
-                "Piped supply outside the building/ Public tap/standpipe",
-                "Tube well / Borehole /Shallow tubewell/borehole",
+                "Piped supply inside the building / Piped water into dwelling (into the building)",
+                "Piped supply outside the building / Public tap/standpipe",
+                "Tube well / Borehole / Shallow tubewell/borehole",
                 "Protected dug well",
                 "Protected spring",
                 "Bottled water",
-                "Rain water",
+                "Rainwater",
```

Comment:
do we have code here? is this list working? I noticed that the other question has list of dict, but this one is list of string

Reply:
Hi mas @dedenbangkit, those are list options of dependency, not the question so its works without adding code and as a list of strings.